migrations/rst/sql/v1.0.12__recreation_map_feature.sql

```diff
@@ -0,0 +1,95 @@
+create table rst.recreation_map_feature
+(
+    rmf_skey                    integer not null
+        primary key,
+    rec_resource_id             varchar(10) references rst.recreation_resource,
+    section_id                  varchar(30),
+    amendment_id                integer,
+    amend_status_code           varchar(3),
+    recreation_map_feature_code varchar(3),
```

Comment:
```suggestion
    recreation_map_feature_code varchar(3) not null references rst. recreation_map_feature_code (recreation_map_feature_code)
```

No feature code table?